src/Domain/Donation.php

```diff
@@ -344,55 +348,44 @@ public function getDonorLastName(): ?string
         return $this->donorLastName;
     }
 
-    /**
-     * @param string $donorLastName
-     */
-    public function setDonorLastName(string $donorLastName): void
+    public function setDonorLastName(?string $donorLastName): void
     {
         $this->donorLastName = $donorLastName;
     }
 
-    public function getDonorPostalAddress(): ?string
+    public function getDonorBillingAddress(): ?string
     {
         return $this->donorPostalAddress;
```

Comment:
@NoelLH probably should look to update these so it matches the method

Reply:
@tbg-jasonfung the property is left as-is intentionally because Doctrine ORM annotations use it to determine the database field name. Changing or customising that would mean more cognitive overhead. As the getter/setter are the external interface for the field I don't think it's a problem to clarify their names while leaving the field as-is.